minio/minioadmin.py

```diff
@@ -48,9 +48,10 @@ def _run(self, args, multiline=False):
             capture_output=True,
             timeout=self._timeout,
             check=True,
+            text=True,
         )
```

Comment:
Add a fix generically to avoid any error on `json.loads()`  here
```py
if not proc.stdout:
    return [] if multiline else {}
```

Reply:
Ok